tests/Aspire.Hosting.Azure.Tests/AzureStorageEmulatorFunctionalTests.cs

```diff
@@ -146,16 +145,16 @@ public async Task VerifyAzureStorageEmulator_blobcontainer_auto_created()
         using var app = builder.Build();
         await app.StartAsync();
 
+        var rns = app.Services.GetRequiredService<ResourceNotificationService>();
+        await rns.WaitForResourceHealthyAsync(storage.Resource.Name, CancellationToken.None);
```

Comment:
Why can't we wait for the `blobContainer` resource?

Reply:
Chatted offline and this is the reason. The health checks were added back on the child resources because the check is not applied on parent resources like WaitFor is doing.